lib/src/widgets/chat/ds_message_bubble.widget.dart

```diff
@@ -16,6 +19,9 @@ class DSMessageBubble extends StatelessWidget {
       vertical: 8.0,
       horizontal: 16.0,
     ),
+    this.shouldUseDefaultSize = false,
```

Comment:
Shouldn't this be ```true``` by default so all bubble related widgets would also have this pixel-constraint?